src/components/shared/Fields/InputLabel/InputLabel.tsx

```diff
@@ -68,9 +69,11 @@ const InputLabel = ({
       id={inputId ? `${inputId}-label` : undefined}
       htmlFor={inputId || undefined}
     >
-      <span className={styles.labelText}>{labelText}</span>
-      {helpText && <span className={styles.help}>{helpText}</span>}
-      {extra && <span className={styles.extra}>{extra}</span>}
+      <div>
+        <span className={styles.labelText}>{labelText}</span>
+        {helpText && <span className={styles.help}>{helpText}</span>}
+      </div>
```

Comment:
Why was this wrapped inside a block ?

Reply:
@rdig This was a change I made when doing the mobile styles in the dapp. It's to be able to use `flex`'s `space-between` to align the label elements, which made correct alignment automatic without need for margins/padding. It ensures you get this:
![existing](https://user-images.githubusercontent.com/64402732/203328292-32763169-8ba3-4a66-8385-2a93c36e3786.png) Instead of this:
![space-between](https://user-images.githubusercontent.com/64402732/203328736-3b9a9703-b085-45fd-a383-05bc3464654c.png)

Reply:
Right. I wasn't "accusing" you of this being a wrong change, I was just curious as to why.

All good then.